core/server/worker/src/main/java/alluxio/worker/block/DefaultBlockWorker.java

```diff
@@ -524,41 +528,48 @@ public static final class Metrics {
     /**
      * Registers metric gauges.
      *
-     * @param blockWorker the block worker handle
+     * @param blockworker the BlockWorker
      */
-    public static void registerGauges(final BlockWorker blockWorker) {
-      MetricsSystem.registerGaugeIfAbsent(
+    public static void registerGauges(final BlockWorker blockworker) {
+      // AtomicReference<BlockMetaMetricCache> m = blockWorker.getBlockMetaMetricCache();
+      CachedGauge<BlockWorkerMetrics> cache = new CachedGauge<BlockWorkerMetrics>(1000, TimeUnit.MILLISECONDS) {
```

Comment:
@jiacheliu3 do you have suggestions for the metrics update interval?

Reply:
we don't need a key for this very specific metric. I think since this is a cache, it should have a longer interval than a normal Prometheus probe interval. Something like 5s will be very reasonable.